tests/requirements.txt

```diff
@@ -1,4 +1,3 @@
-dash==0.38.0
+dash[testing]>=1.0.2
 pytest
```

Comment:
no need to have pytest relevant dependencies here as `dash[testing]` will control that